apps/cowswap-frontend/src/common/pure/TradeDetailsAccordion/index.tsx

```diff
@@ -14,7 +14,26 @@ interface TradeDetailsAccordionProps {
   children?: ReactNode
   open: boolean
   onToggle: () => void
-  feeWrapper?: (feeElement: ReactNode) => ReactNode
+  feeWrapper?: (feeElement: ReactNode, isOpen: boolean) => ReactNode
+}
+
+function createDefaultFeeContent(
```

Comment:
It's better to avoid render functions and use components instead

Reply:
Addressed